src/Tests/Microsoft.NET.Perf.Tests/FolderSnapshot.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Microsoft.NET.Perf.Tests
+{
+    class FolderSnapshot : IDisposable
+    {
+        string OriginalPath { get; set; }
+        string BackupPath { get; set; }
+
+
+        public static FolderSnapshot Create(string path)
+        {
+            FolderSnapshot ret = new FolderSnapshot();
+
+            ret.OriginalPath = path;
+            ret.BackupPath = Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString());
+
+            MirrorFiles(ret.OriginalPath, ret.BackupPath);
+
+            return ret;
+        }
+
+        public void Restore()
+        {
+            MirrorFiles(BackupPath, OriginalPath);
+        }
+
+        public static void MirrorFiles(string source, string dest)
+        {
+            if (Directory.Exists(dest))
+            {
+                Directory.Delete(dest, true);
+            }
+            CopyDirectory(source, dest);
```

Comment:
NIT: empty line after closing brackets. Check this in the rest of the files.

We should add this to our editorconfig.

Reply:
Fixed...